packaging/nuget/nservicebus.testing.fakes.nuspec

```diff
@@ -20,7 +20,7 @@
   <files>
     <file
       src="..\..\src\NServiceBus.Testing.Fakes\*.cs"
-      target="content\Testing\Fakes"
+      target="content\App_Packages\NSB.Testing.Fakes.$version$"
```

Comment:
as stated in the description I didn't use the id intentionally since we're not doing this on the other source packages and I'd rather have internal consistency. But I have no strong opinion on this and I'm happy to discuss this, but then we need to change ATT and Container Sources too.